packages/glibc_lib237.rb

```diff
@@ -77,7 +77,9 @@ def self.postinstall
       if File.file?(File.join(CREW_LIB_PREFIX, 'libC.so.6'))
         Dir.chdir(CREW_LIB_PREFIX) do
           libc_patch_libraries.each do |lib|
-            Kernel.system "patchelf --add-needed libC.so.6 #{lib}" and Kernel.system "patchelf --remove-needed libc.so.6 #{lib}"
+            FileUtils.cp lib, "#{lib}.tmp"
+            Kernel.system "patchelf --add-needed libC.so.6 #{lib}.tmp" and Kernel.system "patchelf --remove-needed libc.so.6 #{lib}.tmp"
+            FileUtils.mv "#{lib}.tmp", lib
```

Comment:
Why is a `.tmp` file necessary?

Reply:
Maybe running patchelf on a running binary was causing segfaults....